package.json

```diff
@@ -63,7 +63,7 @@
     "body-parser": "1.20.0",
     "dateformat": "3.0.3",
     "express": "4.18.1",
-    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#mru_sendUpdatefromthescratch",
```

Comment:
Rollback this once PR https://github.com/telefonicaid/iotagent-node-lib/pull/1513 gets merged before merging this PR.

Reply:
```suggestion
    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
```